types/electron-store/index.d.ts

```diff
@@ -79,12 +79,12 @@ declare class ElectronStore<T = {}> implements Iterable<[string, JSONValue]> {
      */
     onDidChange<K extends keyof T>(
         key: K,
-        callback: (newValue: T[K], oldValue: T[K]) => void
-    ): void;
+        callback: (newValue: T[K] | undefined, oldValue: T[K] | undefined) => void
+    ): () => void;
```

Comment:
AFIR, `onDidChange` returns an arrow function wrapping a call to the `removeListener` function from an EventEmitter instance that the `conf` instance is using. When called, that returned function returns a reference to the `EventEmitter` instance, not `void`.

Reply:
You right! I've corrected this issue